server/src/mappers/equipment/Equipment.ts

```diff
@@ -0,0 +1,19 @@
+import { Equipment } from "../../models/equipment/equipment";
+
+export function equipmentToDomain(raw: any): Equipment[] {
+  return raw.map((i: any) => {
```

Comment:
Do we need to consider the possibility of `raw` being `null` or `undefined`?

Reply:
@MatthewGalan it calls a function `singleEquipmentToDomain` which takes care of that

Reply:
Right, but wouldn't we get an error like `Cannot read properties of undefined (reading 'map')` on line 4 first?